### PR TITLE
[DOC] Add information about Google Java Formatter

### PR DESCRIPTION
**Pull request by m273d15**
* Added IDE specific information
* Wrote example git pre-commit hook

### CODE REVIEW

File: docs/contribute/development-environment.md [hunk 1]
```diff
@@ -68,3 +76,54 @@ In order to build the whole project without using existing build artifacts simpl
...
+  set -e
+  curl -Lo $jar_path "$url" > /dev/null
+  set +e
+}
+
+# check whether formatter already exist
+[ ! -f "$formatter_jar_path" ] && download_formatter "$formatter_jar_path"
+
+# get staged files
+files_to_check=`git diff --cached --name-only | grep "^.*.java$" | tr '\n' ' '`
```
Comment (stefaus) [inline docs/contribute/development-environment.md]: you should filter delete files here `--diff-filter=d`

**Comment by tobous** [comment]
The PR build seems to have some kind of problem. I tried to restart it in travis but it failed with
```
Could not fetch .travis.yml from GitHub.
/home/travis/build.sh: line 2: travis_terminate: command not found
/home/travis/build.sh: line 3: travis_cleanup: command not found
/home/travis/build.sh: line 4: travis_footer: command not found
```
Maybe this is a temporary Travis/Github issue?

**Comment by m273d15** [comment]
@stefaus Could review this PR again?

**Comment by stefaus** [comment]
@m273d15 my remark is still unaddressed
in [guidelines.md#configure](https://github.com/saros-project/saros/blob/pr/doc/formatting/docs/contribute/guidelines.md#configure):
`Import the formatter profile: ... "formatter-profile.xml" `

**Comment by m273d15** [comment]
Thank you I forgot to change this part of the documentation. I moved the useful part into the development environment documentation.

**Pull request merged**